app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less

```diff
@@ -36,6 +36,17 @@
             min-height: 50px;
         }
     }
+
+    .page-actions {
+        > button {
+            float: right;
+        }
+        > button#back {
+            float: left;
+            position: absolute;
+            right: 348px;
```

Comment:
Please, avoid using hardcoded values. In that way, we introduce a workaround but not a solution. 
Try viewing the page with ~1005px screen width. The "back" button behaves better now but still "jumping"

Reply:
@magento-engcom-team give me test instance

Reply:
we use variable against hardcoded value. also now it's looking good for ~1005px screen width.
@rogyar  Please review.